plugins/com.google.cloud.tools.eclipse.appengine.deploy/META-INF/MANIFEST.MF

```diff
@@ -21,4 +21,5 @@ Require-Bundle: com.google.cloud.tools.app.lib;bundle-version="0.1.0",
  org.eclipse.wst.common.modulecore,
  org.eclipse.ui.console,
  com.google.cloud.tools.eclipse.sdk,
- org.eclipse.e4.core.contexts
+ org.eclipse.e4.core.contexts,
+ com.google.cloud.tools.eclipse.util
```

Comment:
I just want to point out that the order of the `Require-Bundle` matters as it forms the bundle's classpath.  I'd always place `org.eclipse.core.runtime` last (but [I'd try to replace it](https://github.com/GoogleCloudPlatform/gcloud-eclipse-tools/pull/386#discussion_r71262501)) too